dwr-hecdssvue/src/main/java/gov/ca/dwr/hecdssvue/panel/MonthlyTablePanel.java

```diff
@@ -153,7 +153,13 @@ public MonthlyTablePanel(Frame frame) {
 		JLabel precisionLabel = new JLabel("Decimal Places: ");
 		add(panel, "North");
 		// _table = new MonthlyTable(this);
-		setShowCommas(DssPluginCore._preferences.getBoolean(SHOW_COMMAS, true));
+		System.out.println("About to set the preferences.");
```

Comment:
Is this source code modification intentional?

Reply:
The null check on the preferences has been removed. This was tied to an issue with the Activator not being started which has been resolved.